samples/basic/clock_pwm/boards/xmc45_relax_kit.overlay

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2023 Andriy Gelman
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	pwmclock: pwmclock {
+		status = "okay";
+		compatible = "pwm-clock";
+		#clock-cells = <1>;
+		pwms = <&pwm_ccu40 2 PWM_HZ(1000000) PWM_POLARITY_NORMAL>,
+		       <&pwm_ccu40 3 PWM_HZ(4000000) PWM_POLARITY_NORMAL>;
+	};
+
+	samplenode: samplenode {
+		status = "okay";
+		compatible = "example-node";
```

Comment:
I suggest to use [`zephyr,user` compat](https://docs.zephyrproject.org/latest/build/dts/zephyr-user-node.html#the-zephyr-user-node)